javascript/lesson4/exercise1/main.js

```diff
@@ -0,0 +1,17 @@
+function exercise1(){
+    var inventaire=["salade", "tomate", "oignons"];
+    console.log( inventaire.length);
+    console.log(inventaire[0]);
+    inventaire.push("sauce blanche");
+   console.log( inventaire.length);
+    inventaire.pop("sauce blanche");
+    afficher(inventaire);
+
+}
+
+exercise1();
+
+    function afficher(tab) {
+        for var i=0;i<inventaire.lenght; i++) 
```

Comment:
il manque une parenthèse après le for
for(var i = 0; ....)
et length, pas lenght